### ISSUE TITLE
Call interactively

### ISSUE THREAD

**Issue by strickinato** [magnars/s.el#115]
Hey everyone,

I'm interested in using these functions interactively.

In a dream world, I'd love to bind these to some sweet hotkeys, mark a region and be able to quickly camelCase some text. Basically, I'd like the exact same flow as `evil-upcase`, but with all of the amazing functions provided by s.el.

I'm looking for guidance in one of the following ways:

### Make a PR?
I imagine if I were to make a PR, it would involve implementing a helper for each of these using `call-interactively` like in https://www.gnu.org/software/emacs/manual/html_node/elisp/Interactive-Call.html

That seems inefficient / not how it should be done

### Use another package?
Maybe there is something else out there that does this already for me. Maybe you all know of this package.

### Something I'm not thinking of?

<3 Side note: I'm a spacemacs user <3

**Comment by magnars** [comment]
Yes please, I want this too. :) This has been discussed before, but my initial dig has not uncovered any traces of it. I guess that's one problem with having such a short package name, huh.

It should be possible to do some macro shenanigans to define a whole bunch of these in one go. It definitely needs to be a separate package tho.

**Comment by strickinato** [comment]
Thanks for getting back at me! I was able to achieve mostly what I wanted by wrapping your work up in some evil commands:

https://github.com/strickinato/evil-briefcase

I'm not sure how useful it will be to the emacs community at large, but I'm excited about it for those of us who use evil (which I guess is quite a few given spacemacs' popularity).

It's my first foray into emacs lisp, so if you're feeling super generous, I'd be interested in some feedback. I did spend a little time diving into "macro shenanigans", but got lost in the frustrations of what to quote and what not to, so I punted there.

Feel free to close the issue if you'd like. I feel satisfied, but I'll leave it up to you to decide if this is still something you want outside of evil and how discoverable you want this to be.

Thanks for inspiring me to dive in myself!! And also thanks for the emacs rocks videos! They're great
